### ISSUE TITLE
--fix incorrectly removes closing brace of block if followed by semicolon {};

### ISSUE THREAD

**Issue by sparr** [palantir/tslint#3393]
### Bug Report

- __TSLint version__: 5.0.0-dev.0
- __TypeScript version__: 2.7.0-dev.20171025
- __Running TSLint via__: CLI

#### TypeScript code being linted

```ts
{{
  }; }
```
(no trailing newline)

with `tslint.json` configuration:

```json
{"extends" : ["tslint:recommended"] }
```

#### Actual behavior

stdout:
```
Fixed 1 error(s) in test.ts

ERROR: test.ts[2, 5]: file should end with a newline
```

resulting file:
```ts
{{
 ; }
```
(still no trailing newline)

#### Expected behavior

newline added at the end, `}` not mysteriously (and syntax-error-inducingly) deleted

**Comment by ajafff** [comment]
Please update to TSLint 5.8.0
We fixed a problem with `--project --fix` that likely causes your issue.

**Comment by ajafff** [comment]
This was fixed by #2379 and released as with TSLint 5.0.0
